integration/tests/basic/test_contract_reverting.py

```diff
@@ -134,3 +137,31 @@ def test_custom_error_revert(self, revert_contract):
     def test_assert_revert(self, revert_contract):
         with pytest.raises(web3.exceptions.ContractPanicError, match="Panic error 0x01: Assert evaluates to false"):
             revert_contract.functions.doAssert().call()
+
+    @pytest.mark.slow
+    @pytest.mark.timeout(40 * Time.MINUTE)
+    def test_reverted_tx_receipt_after_30_minutes(self, revert_contract, json_rpc_client):
+        sender_account = self.accounts[0]
+        # instruction_tx = revert_contract.functions.doTrivialRevert().build_transaction(
+        instruction_tx = revert_contract.functions.customErrorRevert(1, 2).build_transaction(
+            {
+                "from": sender_account.address,
+                "nonce": self.web3_client.eth.get_transaction_count(sender_account.address),
+                "gasPrice": self.web3_client.gas_price(),
+                "gas": 100000000,
+            }
+        )
+        receipt = self.web3_client.send_transaction(account=sender_account, transaction=instruction_tx)
+
+        method = "eth_getTransactionReceipt"
+        params = [receipt["transactionHash"].hex()]
+        response_json_rpc = json_rpc_client.send_rpc(method=method, params=params)
+        response_web3_client = self.web3_client.eth.get_transaction_receipt(receipt["transactionHash"].hex())
```

Comment:
Could you let me know if you asked Andrew about this behavior? I think we don’t need this case (check slack discussion) @kristinaNikolaevaa

Reply:
@kristinaNikolaevaa what you think, remove?

Reply:
yes, we can add this test very fast when the expected behavior will be understandable

Reply:
Removed.